src/views/Transitioner.js

```diff
@@ -155,7 +155,11 @@ class Transitioner extends React.Component<*, Props, State> {
     const { timing } = transitionSpec;
     delete transitionSpec.timing;
 
-    const animations = indexHasChanged
+    const toValue = nextProps.navigation.state.index;
+    const positionHasChanged = position.__getValue() !== toValue;
```

Comment:
This `__getValue()` function will not provide the correct value when the animated value is on the native thread.

Instead, we can use the asynchronous `animatedValue.stopAnimation(value => doSomethingWith(value));`